procedures/foreman/branch.md.erb

```diff
@@ -74,10 +68,10 @@
 
 ## Release Engineer
 
-- Branch RPM packaging using [tool_belt](https://github.com/theforeman/tool_belt)
-  - [ ] Clone tags and create build targets in Koji: `bundle exec ./tools.rb koji --commit configs/foreman/<%= release %>.yaml`
-    Note that you must use koji client < 1.30 since clone-tag was changed and our Koji server is too old. See https://docs.pagure.org/koji/release_notes/release_notes_1.30/#system-changes for details.
+- Branch RPM packaging
   - [ ] Create a `rpm/<%= release %>` branch in [foreman-packaging](https://github.com/theforeman/foreman-packaging) based on `rpm/develop`
```

Comment:
We should probably make this explicit (which https://github.com/theforeman/theforeman-rel-eng/pull/257 also does for later commands):
```suggestion
  - [ ] Create a `rpm/<%= release %>` branch in [foreman-packaging](https://github.com/theforeman/foreman-packaging) based on `rpm/develop`: `git checkout rpm/develop && git pull && git checkout -b rpm/<%= release %>`
```